tsconfig.json

```diff
@@ -10,6 +10,7 @@
     "outDir": "dist",
     "strict": true,
     "sourceMap": true,
-    "target": "es2018"
+    "target": "es2018",
+    "skipLibCheck": true
```

Comment:
This speeds up the build a bunch, because it only validates our type definitions and not the type definitions in referenced dependencies. This settings is recommended by the TypeScript team.